modules/exploits/multi/misc/bmc_patrol_cmd_exec.rb

```diff
@@ -0,0 +1,1201 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+require 'zlib'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Powershell
+
+  $deflater = $nil
+  $inflater = $nil
+
+  $sboxes = [ 0x20022000,0x20000000,0x0,0x20022000,0x0,0x20022000,0x20000000,0x0,0x20022000,
```

Comment:
Instead of global variables, you should probably try using constants in this case.

Reply:
Resolved